w3d_translator/generateDS/subclasses.py

```diff
@@ -1,22 +1,22 @@
 #!/usr/bin/env python
 
 #
-# Generated Thu Jun  2 10:23:57 2022 by generateDS.py version 2.40.13.
+# Generated Thu Jun 16 13:47:00 2022 by generateDS.py version 2.40.13.
```

Comment:
This looks like a docstring. Can we put this whole comment block all in triple quotes? Or is this all automatically generated?

```python
"""
    Docstring comments
"""
```

Reply:
The file itself is automatically generated so I don't think there's much we can do